test/test_images/sendevent/main.go

```diff
@@ -79,6 +79,15 @@ func main() {
 		maxMsg = m
 	}
 
+	defer func() {
```

Comment:
What is this intended to do? If main() panics, doesn't recovering still exit the program?

Reply:
it logs the error

Reply:
Doesn't Go print that out anyway (along with a stack trace)?

https://play.golang.org/p/tTV9h6Wskq1 has in its output `panic: foobar`

Reply:
I was not getting the error from the pod.